README.md

```diff
@@ -41,28 +46,35 @@ Smoke tests live outside of this repo.
 ## Prerequisites
 
 These test assume you have already set up Keramik.  If you are not also making changes to Keramik itself,
-you will likely want to configure Keramik to always use the 'latest' imageTag.  See comments in k8s/operator/kustomization.yaml
-for instructions on how to do that.
-
+you will likely want to configure Keramik to always use the 'latest' imageTag.  See comments in
+k8s/operator/kustomization.yaml for instructions on how to do that.
 
 ## Environment Variables
 
 The test suite is configured via environment variables.
 
-| Env Var        | Description                                             |
-| -------        | -----------                                             |
-| COMPOSEDB_URLS | Comma separated list of URLs to ComposeDB API endpoints |
-| CERAMIC_URLS   | Comma separated list of URLs to Ceramic API endpoints   |
+| Env Var              | Description                                                        |
+|----------------------|--------------------------------------------------------------------|
+| COMPOSEDB_URLS       | Comma separated list of URLs to ComposeDB API endpoints            |
+| EXTRA_COMPOSEDB_URLS | Comma separated list of URLs to additional ComposeDB API endpoints |
+| CERAMIC_URLS         | Comma separated list of URLs to Ceramic API endpoints              |
+| EXTRA_CERAMIC_URLS   | Comma separated list of URLs to additional Ceramic API endpoints   |
 
 Note, these names use the new API boundaries being currently constructed.
-So confusingly the `js-ceramic` process exposes the `ComposeDB API` while the `rust-ceramic` process exposes the `Ceramic API`.
+So confusingly the `js-ceramic` process exposes the `ComposeDB API` while the `rust-ceramic` process exposes the
+`Ceramic API`.
 
-Eventually we expect the `js-ceramic` repo/process to be renamed to `js-composedb` and the `js-composedb` repo being renamed to `js-compsedb-client` or other similar names.
+Eventually we expect the `js-ceramic` repo/process to be renamed to `js-composedb` and the `js-composedb` repo being
+renamed to `js-compsedb-client` or other similar names.
 
+The `EXTRA_*_URLS` variables are needed because the `*_URLS` are usually injected via `.env` files that are baked into
+the test Docker image, and thus cannot be updated at runtime. The former allow for additional endpoints to be added at
+runtime and used in addition to the formal list of endpoints.
```

Comment:
What is runtime in this context, the cerarmic-test?
Are these urls that are generated as part of the test run?
EXTRA_CERAMIC_URLS is unused at this time?

Reply:
Yeah similar question, can we avoid baking the .env files into the image? Maybe not as part of this change

Reply:
Yeah, when the workflow runs, it calls `port-forward.sh`, which will use `kubectl` to port forward into k8s and expose the ComposeDB API locally. Then these `localhost:X` URLs are injected into the tests via the `EXTRA_*_URLS` into the `ceramic-tests` Docker image. These will get concatenated to the existing `*_URLS`.